src/defs.h

```diff
@@ -222,6 +227,11 @@ enum {
     OPT_RATE_LIMIT,               // 42
     OPT_UC_REUSEADDR,             // 43
     OPT_FULL_RTT,                 // 44
+    OPT_CI_SIG_LVL,               // 45
+    OPT_HISTOGRAM,                // 46
+    OPT_HISTOGRAM_UPPER_RANGE,    // 47
+    OPT_HISTOGRAM_LOWER_RANGE,    // 48
+    OPT_HISTOGRAM_BIN_SIZE,       // 49
```

Comment:
Please consider using `--histogram` option to set all related parameters. See `set_affinity_list` as an example.
I think this manner should look better.
For example: `--histogram bin:lower:upper`